### DISCUSSION TITLE
can I set adaptive volume with this dinosaur of a radio?

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1003] [Q&A]
![image](https://github.com/cedricp/ddt4all/assets/122531259/1161aa30-55cb-4f96-9f23-a25e25690178)

**Comment by AndyIreland** [comment]
it depends what radio ECU shows up when you do a search on CAN2 - then if the parameter is there I think it is called something like speed volume dependent curve or something like that

**Reply by DavidIgor** [reply]
thanks, under what is radio setup? tbd or ?

**Reply by AndyIreland** [reply]
Radio ECU or Navigation ECU
you need specially wired y-type lead or soldered/modified OBD2 reader ELM327 to read CAN2 / CAN HI the radio settings through you car's can socket. 

TDB= Cluster/Dash

**Reply by DavidIgor** [reply]
I have elm327 with switch for can hi . when I scan for ecus with switch on it won't find any... is there a particular way when to switch it?

**Reply by KarelSvo** [reply]
https://github.com/cedricp/ddt4all/discussions/970#discussioncomment-7917859

**Reply by DavidIgor** [reply]
that is what i have!

**Reply by AndyIreland** [reply]
> I have elm327 with switch for can hi . when I scan for ecus with switch on it won't find any... is there a particular way when to switch it?

You sure you have not got the FORSCAN adaptor - normally obd2 ones with switch on are for ford high which the OBD2 is wired differently

**Reply by DavidIgor** [reply]
hmm Ill have to check as I can not see nor set things in radio or mmc

**Reply by DavidIgor** [reply]
@AndyIreland you right! I have one for FORscan (HS can/MS Can) 
I will have to switch cables at that switch... 
right?
